### PR TITLE
Spots Left + Waiting List Message

### PR DESCRIPTION
**Pull request by sbassah**
Original Gist: https://gist.github.com/spivurno/82092c812c49891670dc

### CODE REVIEW

**Review by spivurno — CHANGES_REQUESTED** [review]
1. This is missing an opening <?php tag.
2. Some of the code here appears to be out-of-date. Can you take a look at how the limit and how many left are calculated in [this snippet](https://github.com/gravitywiz/snippet-library/blob/master/gp-limit-choices/gplc-spots-left.php) and use that here?

**Comment by sbassah** [comment]
I have made the changes requested, please.

File: gp-limit-choices/gplc-spots-left-plus-waiting-list.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Gravity Perks // GP Limit Choices // Spots Left + Waiting List Message
+ *
+ * This snippet allows you to display the number of spots left in the label of each choice. If there are no spots left, it will display a waiting list message.
+ * https://gravitywiz.com/documentation/gp-limit-choices/
+ *
+ * Plugin Name:  GP Limit Choices — Spots Left + Waiting List Message
+ * Plugin URI:   https://gravitywiz.com/documentation/gp-limit-choices/
```
Comment (spivurno) [inline gp-limit-choices/gplc-spots-left-plus-waiting-list.php:9]: Let's update this URL to "https://gravitywiz.com/documentation/gravity-forms-limit-choices/"

File: gp-limit-choices/gplc-spots-left-plus-waiting-list.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * Description:  This snippet allows you to display the number of spots left in the label of each choice. If there are no spots left, it will display a waiting list message.
```
Comment (spivurno) [inline gp-limit-choices/gplc-spots-left-plus-waiting-list.php:10]: Let's drop "This snippet allows you to " from the plugin description.

**Review by spivurno — APPROVED** [review]
Approved pending two minor verbiage tweaks. ✅

**Pull request merged**